litigious-liberators/profile_cards/profile_cards.py

```diff
@@ -0,0 +1,210 @@
+        self.popup.content.button.bind(on_press=self.popup.dismiss)
+
+    def on_press(self):
+        self.popup.open()
+
+    def close_popup(self):
+        self.popup.dismiss()
+
+
+class SelectionScreen(Screen):
+    def on_pre_enter(self, *args):
+        self.ids["profile_list"]._keyboard = Window.request_keyboard(
+            self.ids["profile_list"]._keyboard_closed, self.ids["profile_list"]
+        )
+        self.ids["profile_list"]._keyboard.bind(
+            on_key_down=self.ids["profile_list"]._on_keyboard_down
+        )
+        return super().on_pre_enter(*args)
+
+
+class ReplayScreen(Screen):
+    def __init__(self, **kw):
+        super().__init__(**kw)
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        sourcefile = os.path.join(dir_path, "./winloss.jpg")
+        self.children[0].children[-1].source = sourcefile
+
+    def replay(self):
+        profile_carousel = self.manager.get_screen("create_profile").ids.profile_carousel
+        profile_carousel.index = 0
+        self.manager.current = "create_profile"
+
+
+class LossScreen(ReplayScreen):
+    pass
+
+
+class WinScreen(ReplayScreen):
+    pass
+
+
+class ProfileCard(Screen):
+    def __init__(self, profile, **kwargs):
+        super(ProfileCard, self).__init__(**kwargs)
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        pic_addr = f"../profiles/pictures/{profile['Picture']}"
+        sound_addr = f"../profiles/sounds/{profile.get('Sound')}"
+        self.ids.picture.source = os.path.join(dir_path, pic_addr)
+        self.sound = SoundLoader.load(os.path.join(dir_path, sound_addr))
+        self.deltas = Counter(profile["Deltas"])
+        self.ids.name.text = profile["Name"]
+        self.ids.right_choice.text += profile["Choices"]["right"]
+        self.ids.left_choice.text += profile["Choices"]["left"]
+        self.results = profile["Results"]
+        self.ids.picture.popup.content.story.text = profile["Biography"]
+        # self.ids.name.font_name = "../fonts/Oldenburg/Oldenburg-Regular.ttf"
+        # #When launching it from terminal using main.py the relative path to font changes.
+        # To test this part as isolated app uncomment the line above and comment the line bellow
+        self.ids.name.font_name = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "../fonts/Oldenburg/Oldenburg-Regular.ttf"
+        )
+
+    def on_enter(self, *args):
+        if self.sound:
+            self.sound.play()
+        return super().on_enter(*args)
+
+
+class ProfileList(ScreenManager):
+    _attributes = DictProperty(None)
+
+    def __init__(self, **kwargs):
+        super(ProfileList, self).__init__(**kwargs)
+        self.profile_dir = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "../profiles/write-ups"
+        )
+        help = Label(
+            text="""Your goal is to maximize all three attributes: Knowledge, Welfare,Energy:
```

Comment:
Grammar and dialect of this label are a bit weird, instructions should be clear, not personalised, which means no `;)`, spaces after commas and `arrow keys`